src/main/groovy/de.flachnetz.golang/GoPlugin.groovy

```diff
@@ -119,78 +125,84 @@ class GoPlugin implements Plugin<Project> {
                     commandLine "rm", "-rf", "build/", config.binaryName, gopath
                 }
 
-                project.task('golang') << {
-                    ensureGoTools(goroot, config.goVersion)
-                }
+                if (!project.parent) {
```

Comment:
Tasks like golang, gopath etc we only need once. You can refer to them by their absolute name: ":gopath"